klock/src/commonMain/kotlin/com/soywiz/klock/locale/ru.kt

```diff
@@ -1,13 +1,23 @@
 package com.soywiz.klock.locale
 
 import com.soywiz.klock.DayOfWeek
+import com.soywiz.klock.KlockLocalContext
 import com.soywiz.klock.KlockLocale
+import com.soywiz.klock.KlockLocaleGender
 
 val KlockLocale.Companion.russian get() = RussianKlockLocale
 
 open class RussianKlockLocale : KlockLocale() {
 	companion object : RussianKlockLocale()
 
+    override fun getOrdinalByDay(day: Int, context: KlockLocalContext): String = when (context.gender) {
+        KlockLocaleGender.Masculine -> "$day-й"
+        // if feminine is ever added to KlockLocaleGender, don't forget to add implementation here: "$day-я"
```

Comment:
Let me be sure I understand: 

Are there there three ways for doing this?

```
Masculine: $day-й
Femenine: $day-я
And neutral: $day-е
```

Is that the case? And if it is, why aren't you adding femenine here too? Is it because this form won't happen on 1-31 days?

Reply:
Exactly. I just couldn't find an example where this ordinal is used in femenine for "date numbers" in Russian. So I think it's redundant to add femenine to enum for now.

If femenine is added by another language or someone finds such an example in Russian, I think it'll be a right rime to add it.